.github/workflows/test.yml

```diff
@@ -26,17 +26,17 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - ruby: "2.4"
```

Comment:
dropping ruby 2.4 would be a breaking change. I want to discuss dropping 2.4/2.5/2.5 support tomorrow at cfgmgmtcamp.

Reply:
Sounds good. `URI#open` is weirdly a private method in 2.4 but we need it in >= 3.0, I'll find a way around it.